Packs/FireEyeNX/Classifiers/classifier-FireEye_NX.json

```diff
@@ -0,0 +1,17 @@
+{
+ "name": "FireEye NX - Classifier",
+ "type": "classification",
+ "id": "FireEye NX",
+ "description": "Classifies FireEye NX Incidents.",
+ "defaultIncidentType": "FireEye NX",
```

Comment:
This incident type doesn't exist anymore - here you need to replace this to one of the incident types you have or create a new one.